docs/docs/reference/crd-reference/config.md

```diff
@@ -39,17 +40,32 @@ spec:
       to put into the same auto-generated [KeptnApp](app.md).
       The default value is 30 (seconds).
     * **cloudEventsEndpoint** -- Endpoint where the lifecycle operator posts Cloud Events.
+    * **blockDeployment** -- If set to `true` (default), application deployment is blocked until the
+      pre-deployment tasks and evaluations succeed.
+      You can set this field to `false` when building up
+      your pre-deployment tasks and evaluations
+      so that your application is deployed
+      even if the pre-deployment tasks and/or evaluations fail.
+      For more information see the
+      [non-blocking deployment section](../../components/lifecycle-operator/keptn-non-blocking.md).
 
 ## Usage
 
 Each cluster should have a single `KeptnConfig` CRD that describes all configurations for that cluster.
 
 ## Example
 
-### OTel example
+This example specifies the URL of the OpenTelemetry collector (1)
+and that the automatic app discovery (2) should be run every 40 seconds.
+Additionally the CloudEvents endpoint URL (3) is specified and the
+blocking of the deployment of the application is disabled (4) in case
+of the pre-deployment task or evaluation failure.
```

Comment:
```suggestion
This example specifies:
- The URL of the OpenTelemetry collector (1)
- The automatic app discovery (2) should be run every 40 seconds
- The CloudEvents endpoint URL (3)
- The blocking of the deployment of the application is disabled (4) in case
of the pre-deployment task or evaluation failure.
```
I like this is a bullet list.  I'm not sure if it should be coded with - or * but `make markdownlint-fix` will correct it.

In the preview, your numbers are lost -- displaying a little arrow in a circle.  Any idea why?

Reply:
that's the point we wanted to achieve, it's called annotations and it's part of mkdocs :) https://squidfunk.github.io/mkdocs-material/reference/annotations/

Reply:
I think what Meg ment is that the list as you make it will become a single paragraph like you se [here](https://keptn--3123.org.readthedocs.build/3123/docs/reference/crd-reference/config/#usage) 

![image](https://github.com/keptn/lifecycle-toolkit/assets/89971034/8aa18640-8b0d-42ce-b279-64744b687b1d)

Reply:
the intention was not to make a list, but just to have a paragraph where we use annotations, so it's easier for the reader to understand what he needs to search for in the example below. Should I make it a list?

Reply:
made it a list

Reply:
I don't see anything that ties the points in your description to the actual sample.  For such a short sample, that may not even be needed.  I was expecting maybe little circled numbers at the appropriate place in the example or something.

Also to consider, if we have some sort of mark displayed in the example, it means that the example can't just be copy/paste'd to start my own yaml file.  I have seen docs where they give the example twice -- once with the numbers for learning comprehension, once without for copy/paste.  It's kind of ugly but we could hide the version for copy/paste under a tab, I suppose.

I like this in a list so it's easier to just scan and see what is described.  Especially here, where the different fields are totally unrelated to each other (from the user perspective) so you aren't "telling a story" or something.